neuralprophet/utils_metrics.py

```diff
@@ -22,21 +22,21 @@ def get_metrics(metric_input):
 
     Returns
     -------
-        dict
-            Dict of torchmetrics.Metric metrics.
+        dict or list
+            Dict of torchmetrics.Metric metrics or list of strings of metrics to use.
     """
     if metric_input is None:
-        return {}
+        return []
     elif metric_input is True:
-        return {k: v for k, v in METRICS.items() if k in ["MAE", "RMSE"]}
+        return ["MAE", "RMSE"]
     elif isinstance(metric_input, str):
         if metric_input.upper() in METRICS.keys():
-            return {metric_input: METRICS[metric_input]}
+            return [metric_input]
         else:
             raise ValueError("Received unsupported argument for collect_metrics.")
     elif isinstance(metric_input, list):
         if all([m.upper() in METRICS.keys() for m in metric_input]):
-            return {k: v for k, v in METRICS.items() if k in metric_input}
+            return metric_input
         else:
             raise ValueError("Received unsupported argument for collect_metrics.")
     elif isinstance(metric_input, dict):
```

Comment:
These user specified metrics will be defined outside and thus not be associated with a device. Might this lead to issues?

Reply:
Resolved in newest commit